please.sh

```diff
@@ -761,6 +761,17 @@ build_and_test_64 () {
 	test $# = 0 ||
 	die "Expected no argument, got $#: %s\n" "$*"
 
+	clean_t='perl -pe "s/ok \d+ # skip/skipped:/" |
+			 perl -pe "s/not ok \d+ -(.*)# TODO known breakage/known e:$1/" |
+			 perl -pe "s/\*\*\* (.+) \*\*\*/$1/g" |
+			 grep -v "^# passed all remaining [0-9]* test(s)" |
+			 grep -v "^# passed all [0-9]* test(s)" |
+			 grep -v "^# still have [0-9]* known breakage(s)" |
+			 grep -v "^1..[0-9]*" |
+			 grep -v "^ok [0-9]" |
+			 perl -pe "s/(.+)/    $1/g" |
+			 perl -pe "s/    ((t\d{4}-|make).+)/$1/g"
```

Comment:
I am not sure if `$1` needs to be escaped here ... too many layers of text insertion 😵